### PR TITLE
Add Electron-React-Boilerplate

### PR DESCRIPTION
**Pull request by airtoxin**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ Some good apps written with Electron.
 
 - [electron-boilerplate](https://github.com/sindresorhus/electron-boilerplate) - Boilerplate to kickstart creating an app.
 - [generator-electron](https://github.com/sindresorhus/generator-electron) - Scaffold out an app boilerplate.
+- [Electron-React-Boilerplate](https://github.com/airtoxin/Electron-React-Boilerplate) - Boilerplate to create Electron app with React.js.
```
Comment (shinnn) [inline readme.md]: to create Electron app -> to create an app

According to the [contribution gudelines](https://github.com/sindresorhus/awesome-electron/blob/ee1614289d523b2c55df84cdfea2a6fa2171c511/contributing.md):

> Don't mention `Electron` in the description as it's implied.

Comment (shinnn) [inline readme.md]: React.js -> React

**Comment by shinnn** [comment]
The boilerplate itself still needs more updates, e.g. [`Atom-Shell` in the package description](https://github.com/airtoxin/Electron-React-Boilerplate/blob/26ea8137c40b364604196ba52b67df2df5f97354/package.json#L4)

**Comment by sindresorhus** [comment]
While the React integration is interesting, the boilerplate itself isn't something I would use or recommend as it contains a lot of IMHO bad patterns and too much boilerplate code. I think this would be more useful as a blog post on how to integrate React into Electron, as the React part is tiny compared to the rest of the content.

**Comment by arthurvr** [comment]
> the boilerplate itself isn't something I would use or recommend as it contains a lot of IMHO bad patterns and too much boilerplate code.

:+1: Interesting concept but more for a blogpost/article/guide IMHO.

**Comment by sindresorhus** [comment]
Happy to link up if you decide to write a blogpost/guide about it ;)

**Pull request closed without merging**